desc/objectives/objective_funs.py

```diff
@@ -1106,42 +1137,67 @@ def print_value(self, *args, **kwargs):
             fmin = jnp.min(f)
             fmean = jnp.mean(f * w) / jnp.mean(w)
 
+            f0 = jnp.abs(f0) if abserr else f0 if val0 is not None else f
```

Comment:
would prefer to bite the bullet and split this up across multiple lines as at least to me it is not super legible. I know what it is doing but the double if on the same line is hard to follow